### DISCUSSION TITLE
Caching in app

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#12842] [Archive]
Hi!

I noticed that, even with caching enabled in the API, the app never serves stuff from cache, since it always sends the `Cache-Control: no-store` header along with every request. ([https://github.com/directus/directus/blob/5cce44fa0547a7e4e6b04e479da245457b238647/app/src/api.ts#L12](https://github.com/directus/directus/blob/5cce44fa0547a7e4e6b04e479da245457b238647/app/src/api.ts#L12))

I was wondering why that choice was made. And if I could create a PR to make that optional?

Cheers!

Kees.

**Comment by benhaynes** [comment]
Intentional, since we always want the App to serve up the latest data. That said, if there's a good use-case to enable _some_ strategic caching, I'd be open to discuss the benefits.
